omegaconf/omegaconf.py

```diff
@@ -620,6 +623,32 @@ def to_container(
             structured_config_mode=structured_config_mode,
         )
 
+    @staticmethod
+    def to_object(
+        cfg: Any,
+        *,
+        resolve: bool = True,
```

Comment:
I think resolve here should not be a member and should be forced to True.

Reply:
Done in https://github.com/omry/omegaconf/pull/502/commits/2bf73b024af397bf60baaf7f713cb3af0c3d4def.